src/TalentConsulting.TalentSuite.Projects.API/Commands/CreateProject/CreateProjectCommand.cs

```diff
@@ -56,7 +56,7 @@ public async Task<string> Handle(CreateProjectCommand request, CancellationToken
             throw;
         }
 
-        if (request is not null && request.ProjectDto is not null)
+        if (request.ProjectDto is not null)
```

Comment:
and if request is null?

request?.ProjectDto (my code is fairly raw tho) |:)

Reply:
If the request is null it will throw an exception at the top of the code. request?.ProjectDto does not really help the exception would have already been thrown